kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.testing.shape;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShapeCollector {
+
+    Shape shape;
```

Comment:
Po co ci to pole? W tej klasie powinieneś mieć tylko listę :-) i metod showFigures() powinna wyświetlać tę listę (iterujemy po liście i dla każdego elementu listy wywołujemy metodę toString())